omero/sysadmins/server-tables.rst

```diff
@@ -90,7 +90,7 @@ After that, the following should succeed:
   your operating system's package installer if possible or if you must
   instead use PyPI_ directly::
 
-    pip install mock
+    pip3 install mock
```

Comment:
FYI need to be careful with `pip` / `pip3`. Sometimes the `3` version is missing from a virtualenv in which case `pip3` might refer to a global pip. Something to check later.